hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOmSnapshot.java

```diff
@@ -416,5 +488,16 @@ private void deleteKeys(OzoneBucket bucket) throws IOException {
       bucket.deleteKey(key.getName());
     }
   }
+
+  private void createKeys(OzoneBucket bucket, String keyprefix) throws IOException {
```

Comment:
We can also use this in ` testListKey ` if possible. Also the method name can be `createKey` instead of `createKeys `since its writing a single key

Reply:
I have renamed the method to createFileKey to avoid conflict with another method. Implemented the method in testListKey